### PR TITLE
Workshop: Navigation

### PR DESCRIPTION
**Pull request by brianegan**
Heya Brett, thanks again for reviewing my workshops! I've got one last workshop on working with Navigator 1, which I've updated the past few days with some tips and tricks I learned while reviewing the other GDE workshops.

Preview link here: https://dartpad.dev/workshops.html?webserver=https://raw.githubusercontent.com/brianegan/codelabs/workshop-nav-1/dartpad_codelabs/src/navigation

### CODE REVIEW

File: dartpad_codelabs/src/navigation/conclusion/snippet.dart [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Home Page'),
+      ),
+      body: ListView.builder(
+        itemCount: 50,
+        itemBuilder: (context, index) {
+          return ListTile(
+            title: Text('Item $index'),
+            // TODO #1: Use pushReplacementNamed to remove the HomePage from the
```
Comment (domesticmouse) [inline dartpad_codelabs/src/navigation/conclusion/snippet.dart:51]: I'd expect that the code shown for the conclusion step shouldn't have TODOs sitting in it. Replace it with the solution from the prior step?

Comment (brianegan) [reply dartpad_codelabs/src/navigation/conclusion/snippet.dart]: Oh my, yes you're right! I thought I'd fixed that, haha. Thanks for the catch :)

**Comment by brianegan** [comment]
Blocked by #522

**Comment by domesticmouse** [comment]
> Blocked by #522

And now #523

**Comment by brianegan** [comment]
Yah sorry, Saturday coding; saw I missed one after the CI failure. Thanks
for patching this one up :)

**Comment by domesticmouse** [comment]
Please update this PR =)

**Comment by brianegan** [comment]
Thanks, @domesticmouse -- updated and awaiting the CI results. One question for ya: I'm preparing the document with links to my workshops & short descriptions. Some of my workshops are hosted on the `https://dartpad-workshops-io2021.web.app` domain, while others are not. For example, [Introduction to Lists](https://dartpad.dev/workshops.html?webserver=https://dartpad-workshops-io2021.web.app/introduction_to_lists) works is now deployed to that domain, but the [Material Design](https://dartpad.dev/workshops.html?webserver=https://dartpad-workshops-io2021.web.app/material_widgets) workshop is not.

Should all of these be accessible via that domain, or should I point all of these workshops to the `raw.githubusercontent.com` domains instead, such as the following link for my material design workshop:

https://dartpad.dev/workshops.html?webserver=https://raw.githubusercontent.com/flutter/codelabs/master/dartpad_codelabs/src/material_widgets

**Pull request merged**